src/scancode/cli.py

```diff
@@ -223,20 +232,39 @@ def reindex_licenses(ctx, param, value):
 number of files processed. Use --verbose to display file-by-file progress.
 '''
 
+formats = ('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')
```

Comment:
I was planning to eventually phase it out as I go on to implement existing formats as plugins, that's why I left it there for now.